index.js

```diff
@@ -5,11 +5,13 @@ const constants = require('./lib/constants');
 const memoizee = require('memoizee');
 
 const appMethod = require('./lib/app');
+const categoriesMethod = require('./lib/categories');
 const getParseList = require('./lib/utils/parseList');
 const parseList = R.partial(getParseList, [appMethod]);
 
 const methods = {
   app: appMethod,
+  categories: categoriesMethod,
```

Comment:
Put the require directly here instead of defining a variable, to follow the convention as the rest of the methods.

Reply:
categories should also be defined in the memoized object below.